src/controllers/trainer.js

```diff
@@ -73,4 +73,31 @@ const updateTrainer = (req, res) => {
     .catch((error) => res.status(500).json(error));
 };
 
-module.exports = { getAllTrainers, getTrainerById, updateTrainer };
+const postTrainer = (req, res) => {
+  const {
+    firstName, lastName, dni, phone, email, city, salary, isActive,
+  } = req.body;
+  trainers.create({
+    firstName,
+    lastName,
+    dni,
+    phone,
+    email,
+    city,
+    salary,
+    isActive,
+  })
+    .then((result) => {
+      res.status(201).json(result);
+    })
+    .catch((error) => {
+      res.status(500).json({
+        message: 'Trainer cannot be created',
+        error,
+      });
+    });
+};
+
+module.exports = {
+  getAllTrainers, getTrainerById, updateTrainer, postTrainer,
```

Comment:
this generated a problem for me (it does not mean that it generated it for you).
to solve it the recommended identation is 
module.exports = {
 getAllTrainers, 
getTrainerById, 
updateTrainer, 
deleteTrainer,
};

Reply:
genial! ahora veo si lo puedo arreglar, porque eso me lo acomodó así el visual, pero bueno saber!